crates/bindings-csharp/Runtime/Runtime.cs

```diff
@@ -11,15 +11,21 @@ public class ReducerContext
     public readonly DateTimeOffset Time;
     public readonly Address? Address;
 
+    /// <summary>
+    /// A reducer-specific instance of `System.Random` that is seeded by current reducer's timestamp. This object is unchanged throught the entire reducer call
+    /// </summary>
+    public Random Rng { get; private set; }
```

Comment:
Since we don't need to set it dynamically anymore, this can be just a readonly field like all the others in this context.

```suggestion
    public readonly Random Rng;
```

Reply:
I don't necessarily agree with with approach, but I changed it for the sake of consistency